Gemfile

```diff
@@ -14,7 +14,6 @@ gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
```

Comment:
It was an attempt to cause a diff in the branches so that I could try and rebase since it didn't go through successfully. However it didn't work.